multicluster/apis/multicluster/v1alpha1/resourceexport_types.go

```diff
@@ -65,6 +69,8 @@ type ResourceExportSpec struct {
 	ExternalEntity *ExternalEntityExport `json:"externalentity,omitempty"`
 	// If exported resource is AntreaClusterNetworkPolicy.
 	ClusterNetworkPolicy *v1alpha1.ClusterNetworkPolicySpec `json:"clusternetworkpolicy,omitempty"`
+	// If exported resource is LabelIdentity of a cluster.
+	LabelIdentity *LabelIdentityExport `json:"labelIdentities,omitempty"`
```

Comment:
the json tag should be "labelIdentity"

Reply:
BTW, not caused by this PR, the json tag of other fields are incorrect, they should use lowerCamelCase. But I'm not sure correcting them directly would affect backward-compatibility and whether we need to guarantee that. @luolanzone

Reply:
I think there will be backward-compatibility issue if we change it directly, we may keep existing fields as it is and guarantee new fields to use lowerCamelCase, @jianjuns any suggestion for this?

Reply:
If we change it directly, I think the object's corresponding fields will be empty for any existing ResourceExports. I will check if there is any code error after upgrade and refine it. Considering it's internal CRDs, we may change it directly with necessary code refinement.